packages/react-server/src/render/error/fallback-error-markup.ts

```diff
@@ -0,0 +1,68 @@
+export const fallbackErrorMarkup = `
```

Comment:
Might be nicer to have this as an actual template, but I know the pathing would be a bit of a pain.

Reply:
yah, i felt that pain and went with this approach instead